miso-dto/src/main/java/uk/ac/bbsrc/tgac/miso/dto/ProgressiveResponseDto.java

```diff
@@ -32,6 +33,14 @@ public long getToken() {
     return updates;
   }
 
+  public boolean isAgain() {
```

Comment:
Seems to indicate that more new data is available than included, but 'isAgain' sounds like it's a repeat of old data. Please rename to clarify meaning